magpie/setup/magpie-setup-project-spark

```diff
@@ -487,7 +487,7 @@ fi
 
 if [ "${ZEPPELIN_SETUP}" == "yes" ]
 then
-    zeppelin_jars="${ZEPPELIN_HOME}/zeppelin-server-${ZEPPELIN_VERSION}-incubating.jar"
+    zeppelin_jars="${ZEPPELIN_HOME}/zeppelin-server-${ZEPPELIN_VERSION}.jar"
     zeppelin_jars="${zeppelin_jars}:${ZEPPELIN_HOME}/lib/*"
     if [ "${sparkclasspath}X" != "X" ]
     then
```

Comment:
could we split this into a separate patch, since it's clearly a bug fix separate from the testsuite additions.  Could be a separate pull request if you want.